generators/app/index.js

```diff
@@ -11,6 +11,21 @@ module.exports = class extends Generator {
     super(args, opts);
   }
 
+  initializing() {
+    if (this.config.existed) {
+      this.log('A Chisel project already exists in this folder.');
+      this.log(
+        'If you are trying to setup existing project check out documentation' +
+          ' at https://www.getchisel.co/docs/setup/wordpress/'
+      );
```

Comment:
Can you change the link to more generic setup page? Also adding some articles, which probably should be there:

_If you are trying to setup an existing project, check out the documentation at https://www.getchisel.co/docs/setup/_